clients/client-accessanalyzer/src/runtimeConfig.ts

```diff
@@ -2,7 +2,7 @@
 // @ts-ignore: package.json will be imported from dist folders
 import packageInfo from "../package.json"; // eslint-disable-line
 
-import { emitWarningIfUnsupportedVersion as awsCheckVersion } from "@aws-sdk/core";
+import { NODE_AUTH_SCHEME_PREFERENCE_OPTIONS, emitWarningIfUnsupportedVersion as awsCheckVersion } from "@aws-sdk/core";
```

Comment:
NODE_AUTH_SCHEME_PREFERENCE_OPTIONS should be imported from `@aws-sdk/core/httpAuthSchemes` to eventually move away from importing anything from the root index of core.